### DISCUSSION TITLE
Reverse Proxy Manager for rtmp

### DISCUSSION THREAD

**Discussion by Zoe8338** [owncast/owncast#1154] [Q&A]
Hey, I am self hosting my owncast instance behind Reverse Proxy Manager and was wondering on how I could set up the reverse proxy of the rtmp protocol to the same domain that I use for the webUI

**Comment by aaronpk** [comment]
I'm not familiar with Reverse Proxy Manager, but you just need to port forward for the RTMP port.

**Reply by Zoe8338** [reply]
You mean port forward on the router?
